config/locales/en.yml

```diff
@@ -338,9 +338,9 @@ en:
       nodes_count:
         one: "%{count} node"
         other: "%{count} nodes"
-      also_part_of_html:
-        one: "part of way %{related_ways}"
-        other: "part of ways %{related_ways}"
+      also_part_of_ways:
+        one: "also part of %{count} way"
+        other: "also part of %{count} ways"
```

Comment:
Should these be `... other way(s)` here? It's not really grammatically correct at the moment given the count exclude one way that the node is a member of.

Reply:
I had it in one of the versions, but then I decided that maybe the messages get too long. Put it back.